solver/src/settlement_submission/flashbots_api.rs

```diff
@@ -69,21 +65,17 @@ impl FlashbotsApi {
 
         match serde_json::from_str::<jsonrpc_core::Output>(&body) {
             Ok(body) => match body {
-                jsonrpc_core::Output::Success(body) => match body.result.as_str() {
+                jsonrpc_core::Output::Success(body) => match body.result.as_bool() {
                     Some(result) => {
-                        tracing::debug!(
-                            "flashbots bundle id: {}",
-                            serde_json::to_string(&result)
-                                .unwrap_or_else(|err| format!("error: {:?}", err)),
-                        );
+                        tracing::debug!("flashbots bundle id: {}", result);
                         Ok(())
                     }
-                    None => Err(anyhow!("result not a string")),
+                    None => Err(anyhow!("result not a bool")),
                 },
                 jsonrpc_core::Output::Failure(body) => Err(anyhow!(body.error)),
             },
             Err(err) => {
-                tracing::error!("flashbot cancellation response: {}", body);
+                tracing::info!("flashbot cancellation response: {}", body);
```

Comment:
I would make this a warn. for info, we try to keep only the short and succinct high-level overviews.

Reply:
Ups, just saw that felix even suggested info. Let's keep it info...